docs/how-to/manage-web-app-rock/index.rst

```diff
@@ -0,0 +1,38 @@
+How to manage a 12-Factor app rock
+**********************************
+
+These guides walk you through all the ways you can manage 12-factor app rocks,
+from initialization to usage.
+
+Initialization
+--------------
+
+Use the pre-defined Rockcraft extension to cater the initial rock files to
+your specific web app framework.
+
+.. toctree::
+    :maxdepth: 2
+
+    Initialize a 12-factor app rock <init-web-app-rock>
+
+Configuration
+-------------
+
+Once your 12-factor app rock has been set up, you can customize many aspects
```

Comment:
I'm wary of "customize" – it could have unintended allusions. Could we instead say "arrange its contents"?